modules/optics/arrow-optics/src/main/kotlin/arrow/optics/dsl/at.kt

```diff
@@ -0,0 +1,84 @@
+package arrow.optics.dsl
```

Comment:
How can this file be accessed, if not from the menu?

Reply:
You mean `DSL` docs?